### PR TITLE
Targets summary table

### PR DESCRIPTION
**Pull request by rpiaggio**
There's a bunch of interactions between the summary table and the tree on the left that we have to take care of.

image

Coming in next PR: attempting to make columns selectable.

### CODE REVIEW

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          )
+          .setHeader("Observations")
+      ).toJSArray
+
+      TargetTableComponent(
+        Table(celled = true, selectable = true, striped = true, compact = TableCompact.Very),
+        header = true
+      )(TargetTable.Options(columns, props.pointingsWithObs.targets.toList.toJSArray))
+    }
+  // .builder[Props]
```
Comment (cquiroz) [inline explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:136]: Commented code?

**Review by cquiroz — APPROVED** [review]
Looks good, I guess it will have more iterations like the idea of filtering Columns, etc

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import reactST.reactTable._
+
+import scalajs.js.JSConverters._
+
+final case class TargetSummaryTable(
+  pointingsWithObs: PointingsWithObs,
+  focused:          View[Option[Focused]],
+  expandedIds:      View[ExpandedIds]
+)(implicit val ctx: AppContextIO)
+// extends ReactProps[TargetSummaryTable](TargetSummaryTable.component)
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:32]: Comment here, too.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**